.claude/skills/prow-ci/fetch_prow_artifacts.py

```
@@ -0,0 +1,217 @@
+
+def download_from_gcs(gcs_path, local_path):
+    """Download a file from GCS using gcloud storage cp."""
+    try:
+        os.makedirs(os.path.dirname(local_path), exist_ok=True)
+        cmd = [
+            'gcloud', 'storage', 'cp',
+            gcs_path,
+            local_path,
+            '--no-user-output-enabled'
+        ]
+        subprocess.run(cmd, check=True, capture_output=True)
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)